R/utils.R

```diff
@@ -19,17 +19,26 @@
 #' )
 #' assay(se)
 #' assay(relativeAb(se))
+#' 
+#' utility function for relativeAb to return counts per million
 #'
+#' @description
+#' This function calculates the relative abundance of each feature in the SummarizedExperiment 
+#' object containing count data, expressed as counts per million (CPM) 
+#' 
+#' @param se returns a new SummarizedExperiment object with counts per million
```

Comment:
Add a #' @param to document the assay argument: "Name or number of the assay of se containing count data to be used for the relative abundance calculation (default: 1L)."

Reply:
This is #' @returns (not param) to document what the function returns.